core/process.py

```diff
@@ -1215,6 +1215,34 @@ def GetLastStopped(self):
     # This be GetCurrent()?  %+ in bash?  That's what 'fg' takes.
     return self.last_stopped_pid
 
+  def WhenContinued(self, pid, waiter):
+    # type: (int) -> None
```

Comment:
OK I think I was confused because this is marked as returning None.

It will fail type checking (and this reminds me that you need a way to push to CI)

Reply:
Is CI the only way get type checking, or can I run this locally? If so, I would add that to my workflow (and also update the wiki / etc to mention how this is done).

Reply:
As an aside, depending on the framework you're using, I could see making a type-level distinction between various integer types:
- status codes
- process ids
- group ids
- user ids 
- file descriptors
etc.